.github/workflows/continuous-integration-cypress.yml

```diff
@@ -0,0 +1,43 @@
+name: Cypress tests
+
+on:
+  workflow_run:
+    workflows: ["Deploy to environment (CIP)"]
+    branches: [ main ]
+    types:
+      - completed
+
+env:
+  NODE_VERSION: 18.x
+
+jobs:
+  cypress-tests:
+    if: github.event.workflow_run.conclusion == 'success' && github.event.workflow_run.inputs.environment != 'Production'
```

Comment:
Thanks for this. Good to see they won't run for prod, providing 'github.event.workflow_run.inputs.environment' works.
I'd be tempted to log it out somewhere here to help confirm it during the testing.